src/sentry/api/endpoints/organization_events_meta.py

```diff
@@ -37,6 +38,56 @@ def get(self, request, organization):
         return Response({"count": result["data"][0]["count"]})
 
 
+class OrganizationEventBaseline(OrganizationEventsEndpointBase):
+    def get(self, request, organization):
+        """ Find the event id with the closest value to an aggregate for a given query """
+        with sentry_sdk.start_span(op="discover.endpoint", description="filter_params") as span:
+            span.set_data("organization", organization)
+            try:
+                params = self.get_filter_params(request, organization)
+            except NoProjects:
+                return Response(status=404)
+            params = self.quantize_date_params(request, params)
+
+        # Assumption is that users will want the 50th percentile
+        baseline_function = request.GET.get("baselineFunction", "p50()")
+        # If the baseline was calculated already save ourselves a query
+        baseline_value = request.GET.get("baselineValue")
+        baseline_alias = get_function_alias(baseline_function)
+
+        with self.handle_query_errors():
+            if baseline_value is None:
+                result = discover.query(
+                    selected_columns=[baseline_function],
+                    params=params,
+                    query=request.GET.get("query"),
+                    limit=1,
+                    referrer="api.transaction-baseline.get_value",
+                )
+                baseline_value = (
+                    six.text_type(result["data"][0].get(baseline_alias))
+                    if "data" in result
+                    else None
+                )
+                if baseline_value is None:
+                    return Response(status=404)
+
+            difference_column = "difference(transaction.duration,{})".format(baseline_value)
+
+            result = discover.query(
+                selected_columns=["timestamp", "id", "transaction.duration", difference_column],
+                # Find the most recent transaction that's closest to the baseline value
+                # id as the last item for consistent results
+                orderby=[get_function_alias(difference_column), "-timestamp", "id"],
+                params=params,
+                query=request.GET.get("query"),
+                limit=2,
+                referrer="api.transaciton-baseline.get_id",
```

Comment:
```suggestion
                referrer="api.transaction-baseline.get_id",
```

Reply:
I'll eventually stop spelling this wrong 🙃